scripts/sct_deepseg_lesion.py

```diff
@@ -51,14 +51,14 @@ def get_parser():
                       mandatory=True,
                       example=['t2', 't2_ax', 't2s'])
     parser.add_option(name="-centerline",
-                      type_value="multiple_choice",
-                      description="choice of spinal cord centerline algorithm.",
+                      type_value="image_nifti",
+                      description="Method used for extracting the centerline.\nsvm: automatic centerline detection, based on Support Vector Machine algorithm.\ncnn: automatic centerline detection, based on Convolutional Neural Network.\nviewer: semi-automatic centerline generation, based on manual selection of a few points using an interactive viewer, then approximation with NURBS.\nprovide the filename of a manual centerline (e.g. t2_centerline_manual.nii.gz).\n",
```

Comment:
This part of the doc is a bit confusing wrt. input file name (it is not clear what one should write after `-centerline` for inputing a file name. I would rewrite as:
~~~
                      type_value="multiple_choice",
                      description="Method used for extracting the centerline.
"\nsvm: automatic centerline detection, based on Support Vector Machine algorithm."
"\ncnn: automatic centerline detection, based on Convolutional Neural Network."
"\nviewer: semi-automatic centerline generation, based on manual selection of a few points using an interactive viewer, then approximation with NURBS."
"\nFILENAME: filename of an existing centerline (e.g. t2_centerline_manual.nii.gz).\n",
~~~
Now, the problem is that my suggestion above will NOT work, because `multiple_choice` will look for the string `FILENAME` instead of an actual file name :-(

I see two alternatives:
- replace `FILENAME` by `manual` and have another flag dedicated to inputing a centerline file, e.g.: `sct_deepseg_sc -i t2.nii.gz -c t2 -centerline manual -filecenterline t2_centerline_manual.nii.gz`. Cons: heavy.
- modify the Parser to have the possibility to not check the input string when using `multiple_choice`. Cons: defeat the purpose of a parsing checker...

@zougloub any idea?

Reply:
@jcohenadad: As you said, I used `type:nifti_input` to allow the user to input a manul centerline. While the other options (i.e. `svm`, `cnn`, `viewer`) are not considered as `Image`, from my understanding, thanks to the parameter ` list_no_image=['svm', 'cnn', 'viewer']`.
--> This approach is inspired from PropSeg:
https://github.com/neuropoly/spinalcordtoolbox/blob/master/scripts/sct_propseg.py#L242

Reply:
Part of the doc corrected as suggested by @jcohenadad, in commit #f34bf6b.

Reply:
the problem is that the way this is currently done in PropSeg is not good either (for the same reasons as here). It would be nice to find a proper solution and then deploy it to the rest of SCT. Waiting for @zougloub's feedback there.

Reply:
@jcohenadad : @zougloub approved the PR, should I merge the branch or wait for further discussions about the point you raised?

Reply:
i suggest the following:
~~~
"svm: automatic centerline detection, based on Support Vector Machine algorithm."
"cnn: automatic centerline detection, based on Convolutional Neural Network."
"viewer: semi-automatic centerline generation, based on manual selection of a few points using an interactive viewer, then approximation with NURBS."
"manual: filename of an existing centerline (e.g. t2_centerline_manual.nii.gz).\n",
~~~
and another flag:
~~~
-file-centerline; type:type:image_nifti; description: Input centerline file (to use with flag -centerline manual)
~~~

Reply:
@jcohenadad : I have made the modifications we discussed yesterday.
Ready to be reviewed.